src/containers/Pages/specials/Index/index.tsx

```diff
@@ -1,17 +1,15 @@
 import ErrorCard from 'containers/Pages/specials/Index/ErrorCard';
 import WorkDoneCard from 'containers/Pages/specials/Index/WorkDoneCard';
 import MembersCard from 'containers/Pages/specials/Index/MembersCard';
-import { useMemberships } from 'api/hooks/Membership';
+import { Groups } from 'types/Enums';
 
 const AboutIndex = () => {
-  const { data } = useMemberships('index');
-  const members = data?.map((member) => `${member.user.first_name} ${member.user.last_name}`);
   return (
     <>
       <WorkDoneCard changelogURL={'https://raw.githubusercontent.com/tihlde/Kvark/dev/CHANGELOG.md'} title={'Hva har vi gjort i frontend?'} />
       <WorkDoneCard changelogURL={'https://raw.githubusercontent.com/tihlde/Lepton/dev/CHANGELOG.md'} title={'Hva har vi gjort i backend?'} />
```

Comment:
Ikke en del av denne pr'en egentlig, men vi bør vise changelog som ligger i master istedenfor dev
```suggestion
      <WorkDoneCard changelogURL={'https://raw.githubusercontent.com/tihlde/Kvark/master/CHANGELOG.md'} title={'Hva har vi gjort i frontend?'} />
      <WorkDoneCard changelogURL={'https://raw.githubusercontent.com/tihlde/Lepton/master/CHANGELOG.md'} title={'Hva har vi gjort i backend?'} />
```